cpp/ql/lib/change-notes/2023-11-8-strsafe-models.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added missing models for strsafe sprintf variants.
```

Comment:
How about something like this instead?

```suggestion
* Added models for the `sprintf` variants from the `StrSafe.h` header.
```

Reply:
Whichever you prefer. I was just capturing that there were existing models there but incomplete, but I'm ok with whatever.